app/controllers/users_controller.rb

```diff
@@ -0,0 +1,35 @@
+class UsersController < ApplicationController
+
+  def index
+    render :template => 'static_pages/home'
+  end
+
+  def show
+    @user = User.find_by id: params[:id]
+    if @user.blank?
+      flash[:danger] = I18n.t "controllers.users_controller.none"
```

Comment:
ở controller thì có em có thể sử dụng I18n một cách ngắn gọn hơn như này nhé:
```suggestion
      flash[:danger] = t "controllers.users_controller.none"
```